src/showcase/app/maps-leaflet/maps-leaflet-examples/leaflet-map-examples/leaflet-map-example/leaflet-map-example.component.ts

```diff
@@ -165,4 +124,44 @@ export class LeafletMapExampleComponent implements AfterViewChecked {
       visible: true,
     });
   }
+
+  /**
+   * The map gets it's size, when it is added to the DOM.
+   * At this point it is not yet displayed, but only when the user clicks the "examples"-tab.
+   * This leads to not enough tiles being loaded when the map is actually displayed, which turns out in a grey map.
+   * All required tiles will not be loaded until you call `map.invalidateSize()`.
+   * Because it is not possible to get notified when the user clicks the "examples"-tab,
+   * we use ngAfterviewChecked to get the new mapsize and load the according tiles.
+   */
+  ngAfterViewChecked() {
+    if (this._map) {
```

Comment:
I noticed that when I am on the example page, navigate to another page (e.g. Use esri-leaflet) and navigate back to the example, it will only be a grey map.

Reply:
I fixed this today and it should work now.